tests/test_api.py

```diff
@@ -811,3 +811,18 @@ def test_query(self):
                 patch.object(MockConnection, 'query') as mock_query:
             self.srv.query('SELECT 1')
             mock_query.assert_called_once_with('SELECT 1')
+
+    def test__construct_server_tokens(self):
+        #
+        # Test cases (case insensitive values):
```

Comment:
```suggestion
```
you need to test not-all-lower-case also works. change at least one value in these use cases

Reply:
Fair point - added tests for mixed cases of each valid value.